src/DependencyInjection/PimcoreGenericDataIndexExtension.php

```diff
@@ -90,11 +91,34 @@ private function registerIndexServiceParams(ContainerBuilder $container, array $
         $definition->setArgument('$systemFieldsSettings', $indexSettings['system_fields_settings']);
 
         $openSearchClientId = 'pimcore.open_search_client.' . $indexSettings['client_params']['client_name'];
+        /**
+         * @deprecated Alias will be removed in version 2.0. Please use "generic-data-index.search-client" instead
+         */
         $container->setAlias('generic-data-index.opensearch-client', $openSearchClientId);
```

Comment:
https://symfony.com/doc/current/service_container/alias_private.html#deprecating-service-aliases

```suggestion
        $container->setAlias('generic-data-index.opensearch-client', $openSearchClientId)
            ->setDeprecated(
                'pimcore/generic-data-index-bundle',
                '1.3',
                'The "%alias_id%" service alias is deprecated and will be removed in version 2.0. Please use "generic-data-index.search-client" instead.'
            );
```

Reply:
thank you